.editorconfig

```diff
@@ -0,0 +1,21 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+root = true
+
+# All files
+[*]
+indent_style = space
+
```

Comment:
I'm not sure that our indentation should be space instead of tab. what's the default?

Reply:
@hassanhabib that is the default.   
When you add it with this option
![image](https://user-images.githubusercontent.com/797750/141698067-b4af6564-c210-4f26-bee0-d45bd7cd0d3e.png)
it looks like this when you open it in a text editor
![image](https://user-images.githubusercontent.com/797750/141698090-149573d4-bc06-4489-9558-931517787567.png)

But I have removed this as as well now so we only have those two settings and none of the defaults.